beacon-chain/flags/archive.go

```diff
@@ -30,3 +30,46 @@ var (
 		Usage: "Whether or not beacon chain should archive historical blocks",
 	}
 )
+
+// ArchiveFlags specifies all the archive flags for the
+// archiver service.
+type ArchiveFlags struct {
```

Comment:
Sorry to be pedantic, I really like this pattern but calling it `ArchiveFlags` is limiting. Can we rename this to global flags?

This is a set of beacon chain permanent global flags. So please add a comment to that effect as well.

Reply:
alright sure, I will rename it. I wanted to do a follow up PR where I convert all flags to globals, so we can do away with the current pattern where we initialize a bool in the service's struct for that flag.